MovingBlocks/Terasology
# fix: remove Rendering Module Settings screen

### Contains

We currently don't have a model that allows us to easily load (parts) of modules to make them more configurable in the game setup screens.

In particular, the `RenderingModuleRegistry` is not yet initialized, and thus the whole purpose of the rendering module configuration is defied.

Therefore, we remove the screen from the menu and also remove the (now dead) code from the repo.

See also [thread on Discord](https://discord.com/channels/270264625419911192/696830442065625118/1038912894097694781).

Resolves to #5075.
Related to #5076.

### How to test

Go through the advanced game setup and ensure that there is no longer an option to configure the rendering module configuration.

### Outstanding before merging

Nothing.
